src/fidesapi/routes/util.py

```diff
@@ -2,7 +2,9 @@
 
 from fastapi import APIRouter
 
-API_PREFIX = "/api/v1"
+from fidesctl.core.utils import API_PREFIX as _API_PREFIX
+
+API_PREFIX = _API_PREFIX
```

Comment:
Was keeping the `API_PREFIX` constant re-defined in here just reduce the changes in the other `fidesapi/routes/` modules or is it a preferred way to do that sort of sharing in the future too? It definitely reduces extra work from the `generate` route we just shipped as well :)